.env

```diff
@@ -193,3 +193,20 @@ MF_POSTGRES_READER_DB_SSL_MODE=disable
 MF_POSTGRES_READER_DB_SSL_CERT=""
 MF_POSTGRES_READER_DB_SSL_KEY=""
 MF_POSTGRES_READER_DB_SSL_ROOT_CERT=""
+
+# Twins
+MF_TWINS_LOG_LEVEL=debug
+MF_TWINS_HTTP_PORT=9021
+MF_TWINS_SERVER_CERT=""
+MF_TWINS_SERVER_KEY=""
+MF_TWINS_DB_NAME=mainflux-twins
+MF_TWINS_DB_HOST=twins-db
+MF_TWINS_DB_PORT=27018
+MF_TWINS_SINGLE_USER_EMAIL=""
+MF_TWINS_SINGLE_USER_TOKEN=""
+MF_TWINS_CLIENT_TLS=""
+MF_TWINS_CA_CERTS=""
+MF_TWINS_MQTT_URL=tcp://mqtt-adapter:1883
+MF_TWINS_THING_ID=8041bbca-34d0-435f-86c1-21798555b59a
+MF_TWINS_THING_KEY=65c0244b-b4ad-4286-a9de-74a19de20b45
+MF_TWINS_CHANNEL_ID=b3cfd2cc-ad49-4ed8-82b7-cb80e544acdb
```

Comment:
we had a discussion about this to remove this things in an .env-twins file as this is not in the standard. What do You think about that?

Reply:
This is planed to be created in the service itself and removed from here. Even if it's in a separate .env it have to be used by the same docker-compose so i don't really see any benefit.

Reply:
Ok